tests/cylc-poll/16-execution-time-limit/suite.rc

```diff
@@ -0,0 +1,20 @@
+[cylc]
+    [[events]]
+        timeout = PT1M
+    [[reference test]]
+        expected task failures = foo.1
+[scheduling]
+    [[dependencies]]
+        graph = """
+            foo
+            foo:fail => ! foo
+        """
+[runtime]
+    [[foo]]
+        script = """
+            #rm "$CYLC_TASK_LOG_ROOT.status"  # Disable polling.
```

Comment:
``#rm`` seems to have been left in here from development/debugging stages.

Reply:
Actually re-running this test the ``rm`` command seems to get run anyway despite the hash preceding it - removing the ``rm`` will make the test throw a ``isodatetime.parsers.ISO8601SyntaxError: Invalid ISO 8601 date representation`` error.

Reply:
I have now removed the `#rm` command.